### ISSUE TITLE
Range Output

### ISSUE THREAD

**Issue by sergegers** [ericniebler/range-v3#1585]
I'm porting a project from boost.ranges v2 to ranges v3 and I 've got a custom stream output for ranges already.  As I know the range printing isn't a part of the standard. Is it possible to disable default range output by conditional compilation or offer the extension machinery for custom output?
Thanks, Serge.

**Comment by marehr** [comment]
https://github.com/ericniebler/range-v3/blob/fa00307bebb503d58107c6abab3bea4ac4930177/include/range/v3/view/interface.hpp#L508-L535

I guess this code conflicts with yours.

**Comment by sergegers** [comment]
Yes, it is. And, due to the output isn't a part of the standard I think to have a way to disable it is a good thing. The similar problem was in ranges v2 but the library looks abandoned and I patched the code myself.
